PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+** Describe the change **
+
+_explantation of what it does_
+
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _`KIALI-7 New icons`_ (Jira-id, one space, description)
+* If this is a fix for a GH-issue, please link it here
+
+** Backwards in compatible? **
+
+[ ] Is your change introducing backwards incompatible changes?
+
+_describe the incomabibilities if appropriate_
```

Comment:
incomabibilities is definitely not a word :)
I propose "please describe in what way the change is incompatible and how does it influence Kiali and its users"